### PR TITLE
Use  highcharts.js

### PR DESCRIPTION
**Pull request by raiyni**
obviously needs refactoring but this is just for the now.sh deploy

it's little bit thicker than chartist but has big data support

### CODE REVIEW

File: src/_data/skills.js [hunk 1]
```diff
@@ -1,3 +1,30 @@
+export const colors = [
```
[the remark this answers is not in the document]
Comment (raiyni) [reply src/_data/skills.js]: Because that is provided as an object not an array

[the remark this answers is not in the document]
Comment (deathbeam) [reply src/_data/skills.js]: Object.keys(skills).map(s => skills[s])

[the remark this answers is not in the document]
Comment (deathbeam) [reply src/_data/skills.js]: But i already do that at other places and it works and the order is staying the same as well

[the remark this answers is not in the document]
Comment (raiyni) [reply src/_data/skills.js]: well that's wrong then and you are lucky it's working

[the remark this answers is not in the document]
Comment (Abextm) [reply src/_data/skills.js]: I'm not aware of any browsers that don't order object keys.

[the remark this answers is not in the document]
Comment (raiyni) [reply src/_data/skills.js]: I think it sorts by int, then string insertion order, then symbol order. Which you order by overall first and then sorted alphabetically. I still wouldn't rely on object key order.

```obj = {
    [Symbol('first')]: true,
    '02': true,
    'ds': true,
    '342h1': true,
    'abc': true,
    [Symbol('second')]: true,
}
Reflect.ownKeys(obj);
(6) ["02", "ds", "342h1", "abc", Symbol(first), Symbol(second)]```

[the remark this answers is not in the document]
Comment (deathbeam) [reply src/_data/skills.js]: well, the object contains only strings as keys and forever will, so, it looks like it is working as expected

**Comment by deathbeam** [comment]
Decided to go with Recharts in #134 as it was a bit nicer than Highcharts (and more integrated, less code to maintain)

**Pull request closed without merging**